webui/react/src/pages/FlatRuns/FlatRunMoveModal.tsx

```diff
@@ -196,7 +220,7 @@ const FlatRunMoveModalComponent: React.FC<Props> = ({
       <RunFilterInterstitialModalComponent
         filterFormSet={filterFormSetWithoutId}
```

Comment:
potential bug: this formset doesn't include the searchid filter. could you check to make sure the interstitial warning does not appear when using `selectAll` on a search's runs where all the runs are valid?